### PR TITLE
👌 Minor okctl.io cleanup

### PR DESCRIPTION
**Pull request by deifyed**
## Description

* Fixes moving from environment to --cluster-declaration leftovers
* Polishes some header structures
* Improves some stuff
* Removes som redundant stuff

## Motivation and Context

[KM253](https://trello.com/c/yfGLpPYz)

[KM205](https://trello.com/c/GPPhDLWh)

A general outdation of docs

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated the release notes (for the next release).

### CODE REVIEW

**Review by ivaruf — APPROVED** [review]
Looks good :+1:

File: userdocs/src/getting-started/application-addons.md [hunk 1]
```diff
@@ -52,18 +56,33 @@ In Kubernetes, it is possible to define a [DNS Policy](https://kubernetes.io/doc
 If you want to attach to the database from intellij or some other IDE, you can do so by setting up a port forwarder, you can do so with the following command:
 
 ```bash
-# uuidgen > my-password-file
-# okctl forward postgres --environment test --name dbtest --username bob --password-file my-password-file
+# Usage
+## Generate a password for the pgBouncer user
+uuidgen > my-password-file
+## Forward traffic from localhost:5432 to the pgBouncer
+okctl --cluster-declaration <path to cluster declaration> forward postgres -name <database server name> --username <pgBouncer username> --password-file my-password-file
```
Comment (yngvark) [inline userdocs/src/getting-started/application-addons.md:63]: Useful improvement!

File: userdocs/src/getting-started/clean-up.md [hunk 2]
```diff
@@ -4,21 +4,23 @@ During testing, or maybe when a project has come to an end - there might be a ne
...
-# okctl delete cluster <environment name>
-#
-# <cluster name>     must equal the environment name used for cluster creation.
-#
+# Usage
+okctl --cluster-declaration <path to cluster declaration> delete cluster
+
 # Example:
-okctl delete cluster test
+okctl --cluster-declaration cluster-yaml delete cluster
```
Comment (yngvark) [inline userdocs/src/getting-started/clean-up.md:11]: `cluster-yaml` - typo? Just use `cluster.yaml` like below?

File: userdocs/src/getting-started/clean-up.md [hunk 3]
```diff
@@ -4,21 +4,23 @@ During testing, or maybe when a project has come to an end - there might be a ne
...
 
 If you want to remove your cluster's primary hosted zone, use
 
 ```bash
-okctl delete cluster <environment name> --i-know-what-i-am-doing-delete-hosted-zone-and-records true 
+# Usage
+okctl --cluster-declaration <path to cluster declaration> delete cluster --i-know-what-i-am-doing-delete-hosted-zone-and-records true 
+
+# Example
+okctl --cluster-declaration cluster.yaml delete cluster --i-know-what-i-am-doing-delete-hosted-zone-and-records true
```
Comment (yngvark) [inline userdocs/src/getting-started/clean-up.md:23]: Haven't we removed the `i-know-what-i-am-doing-delete-hosted-zone-and-records` some time ago? Don't we always delete the hosted zone these days?

File: userdocs/src/getting-started/install.md [hunk 4]
```diff
@@ -1,26 +1,19 @@
...
-### Installation
+## Install on macOS
 
 ```bash
 brew tap oslokommune/tap
 brew install oslokommune/tap/okctl
 ```
 
-## Shell autocompletion
+## Enable Shell autocompletion
```
Comment (yngvark) [inline userdocs/src/getting-started/install.md:17]: Shell -> shell ? Or is it a name?

**Review by yngvark — CHANGES_REQUESTED** [review]
Good to see some doc improvement again.

File: userdocs/src/getting-started/clean-up.md (hunk 2 above)
Comment (deifyed) [reply userdocs/src/getting-started/clean-up.md:11]: my bad

Comment (deifyed) [reply userdocs/src/getting-started/clean-up.md:11]: also, good catch

File: userdocs/src/getting-started/clean-up.md (hunk 3 above)
Comment (deifyed) [reply userdocs/src/getting-started/clean-up.md:23]: oh right. nice catch

File: userdocs/src/getting-started/install.md (hunk 4 above)
Comment (deifyed) [reply userdocs/src/getting-started/install.md:17]: here i'm refering to the gas station

Comment (yngvark) [reply userdocs/src/getting-started/install.md]: Ah then it makes sense. I love autocompletion when I'm at Shell.

**Pull request merged**